### ISSUE TITLE
Possible conflict with Windows Store Python package

### ISSUE THREAD

**Issue by morgansimonsen** [microsoft/winget-cli#580]
# Brief description of your issue
Possible conflict with Windows Store Python package. The productId (9nblggh4nns1) provided for the Windows Package Manager app contains Python binaries.
# Steps to reproduce

Unknown; discovered while trying to install the Windows Package Manager preview after signing up for the preview.

# Expected behavior

That the preview packages is installed and visible under %appdata%\Local\Microsoft\WindowsApps\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe

# Actual behavior
The package is listed under %appdata%\Local\Microsoft\WindowsApps\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe, but contains python.exe and python3.exe.
![image](https://user-images.githubusercontent.com/2271600/94421206-821aba80-0185-11eb-9447-f88d3855c179.png)
![image](https://user-images.githubusercontent.com/2271600/94421194-7e873380-0185-11eb-8ddd-46acd5804541.png)
When performing this steps from the invite email to install the preview package the Windows Store displays this:
![image](https://user-images.githubusercontent.com/2271600/94421446-d756cc00-0185-11eb-8edc-8f186c92755b.png)
The packages also seems "uninstallable":
![image](https://user-images.githubusercontent.com/2271600/94421577-0b31f180-0186-11eb-9f3b-67b6f0181e5b.png)
It can be removed from the command line:
Remove-AppxPackage -Package "Microsoft.DesktopAppInstaller_1.0.32912.0_x64__8wekyb3d8bbwe"
But if I try to reinstall with the instructions from the email, the "Python" package is installed, not the Package manager preview.
If I manually install the appx from the Releases page, it installs itself into the same folder:
![image](https://user-images.githubusercontent.com/2271600/94423063-54834080-0188-11eb-8cb4-bed527973bd5.png)

# Environment
Windows 10 2004 (19041.508)

**Comment by KevinLaMS** [comment]
@morgansimonsen Thanks for the report.  The aliases you see were introduced before the Windows Package Manager.  They were a way to allow developers to type Python on a clean install of Windows and cause a redirect to enable developers to download Python from the store. 

So, currently this is by design.  However, we will investigate what we should do with these aliases in light of the Windows Package Manager CLI (winget.exe).

**Comment by osxdude** [comment]
I followed the process via the Insider welcome email, and experienced the same thing; install App Installer from Store, get `python.exe` and `python3.exe` in the `WindowsApps` folder, which open the respective Store page for Python, as @KevinLaMS stated. I installed the `appxbundle` from the releases page and get `winget.exe` as well as `python.exe` and `python3.exe`.

I believe the problem isn't named correctly; there may not actually be any conflicts, just seems to be no `winget.exe` executable in the package downloaded from the Store.
